### ISSUE TITLE
 Allow a style setting "Above/Below staff" for Figured bass

### ISSUE THREAD

**Issue by MiguelVicenteGarcia** [musescore/MuseScore#24231]
### Your idea

Allowing a style setting "Above/Below staff" (with Below being the default)

### Problem to be solved

Figured bass are currently set below staff, but in practice they appear many times above the staff. Although it is possible to set them above manually, a option for setting "Above/Below staff" for Figured bass is much needed. 
![image](https://github.com/user-attachments/assets/1da9c7e2-3df3-4ddf-ab2c-26e84d7be3c3)

### Checklist

- [X] This request follows the [guidelines](https://github.com/musescore/MuseScore/wiki/Reporting-bugs-and-issues) for reporting issues
- [X] I have verified that this feature request has not been logged before, by searching the [issue tracker](https://github.com/musescore/MuseScore/issues) for similar requests

**Comment by MarcSabatella** [comment]
Unless I'm missing something, there has been no change here - the Property setting has long exists and continues to exist, but there is still no Style setting that I can see?

**Comment by MiguelVicenteGarcia** [comment]
Yes, Marc, you are right.( I mixed up Porperty settings [that I had missed before) and Style settings).

**Comment by Jojo-Schmitz** [comment]
So it isn't really a feature request but a bug report
And came up anain in https://musescore.org/en/node/372825
